src/components/TranslateMarkdown.tsx

```diff
@@ -11,11 +11,11 @@ interface Props {
 const TranslateMarkdown = ({ source }: Props) => {
   return (
     <Markdown
-      disallowedTypes={['html']}
-      renderers={{
-        root: React.Fragment,
-        link: (props) => <LinkApp isExternal={true} {...props} />,
-        paragraph: React.Fragment // Remove <p> added by react-markdown.
+      disallowedElements={['html']}
```

Comment:
Has the API changed here? Why was it bumped?

Reply:
Yup the API has changed and it was bumped to conform to the react version needed by the Library